src/main/scala/com/evolution/scache/Cache.scala

```diff
@@ -348,6 +351,13 @@ trait Cache[F[_], K, V] {
 
 object Cache {
 
+  sealed trait Modification[+F[_], +V]
+  object Modification {
+    final case class Put[F[_], V](value: V, release: Option[F[Unit]]) extends Modification[F, V]
+    final case object Keep extends Modification[Nothing, Nothing]
```

Comment:
please rename to `Ignore` as in case of no `Value`, it's not about keeping it :)